e2e/cookies/CookieManager.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * this class handle the cookies.
+ */
+class CookieManager {
+    constructor() {
+
+    }
+    getInstances(path) {
+
+    }
+
+    /**
+    * return the actual user located into cookie
+    */
+    static getUserCookie(){
+      var userCookie = browser.getCookie('lastuser').value;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "userCookie". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1488)